api/proto/odpf/metadata/optimus/Job.pb.go

```diff
@@ -1,18 +1,18 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.25.0
-// 	protoc        v3.13.0
+// 	protoc-gen-go v1.26.0-rc.1
```

Comment:
can you try running `make install` then generate protos? I think I have pinned the compiler version, not sure why it's generated with a different one.